contracts/leverage_trading/src/metadata.rs

```diff
@@ -168,6 +168,12 @@ pub enum OrderAction {
     Create,
     Cancel,
     Liquidate,
+    Close {
+        order_id: U128,
+        order: Box<Order>,
```

Comment:
Box?

Reply:
Yes. I can't do the reference type because of `BorshDeserialize/BorshSerialize`, and I can't leave the pure type `Order` because the pipelines don't pass